package.json

```diff
@@ -58,13 +58,14 @@
     "express-basic-auth": "^1.2.0",
     "express-ipfilter": "^1.1.2",
     "express-mung": "^0.5.1",
+    "glob": "^7.2.0",
     "isomorphic-fetch": "^3.0.0",
     "mocha-headless-chrome": "^2.0.3",
     "moment": "^2.27.0",
     "normalize-url": "^4.0.0",
     "redis": "^3.1.2",
     "redisscan": "^2.0.0",
-    "universal-analytics": "^0.4.20",
+    "universal-analytics": "^0.5.0",
```

Comment:
As per the package changelog, the changes were just routine maintenance: https://github.com/peaksandpies/universal-analytics/commit/4736a56c234ad67edecbe207cee8bba7e8ff4757